StructuredRAG.Api/appsettings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.EntityFrameworkCore": "Warning"
+    }
+  },
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=sqlserver;Database=StructuredRAG;User Id=sa;Password=YourStrong@Passw0rd;TrustServerCertificate=True;MultipleActiveResultSets=true"
+  },
+  "Gemma": {
```

Comment:
Replace this with: `http://model-runner.docker.internal/engines/llama.cpp/v1`
And rename the Config from Gemma to `DockerModelRunnerUrl`